docs/contributing/building.rst

```diff
@@ -153,6 +153,22 @@ or source it into your shell,
 
    . $BUILD_DIR/python_env.sh
 
+PyTorch
+=======
+
+To install PyTorch, follow the commands below. The first 2 lines install dependencies. The subsequent steps install PyTorch from source.
+
+.. code-block:: bash
+
+   mamba install -c conda-forge --override-channels astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses
+   mamba install -c pytorch magma-cuda110
```

Comment:
Can you create an environment file for this instead of having the list here? Just like `conda_recipe/environment.yml`, but you can put it somewhere more appropriate or rename it.

Reply:
The environment file can also configure the channels as needed.

Reply:
Sure thing, changes reflected